px-typeahead.html

```diff
@@ -115,6 +115,15 @@
         type: String,
         value: ''
       },
+      /**
+       * Headers for pretech request
+       */
+      prefetchHeaders: {
+        type: Object,
+        value: {
```

Comment:
Just recopying my comment in case you didn't see it:

> Also, fun gotcha, it needs to be a function that returns an object otherwise it'll leak into other instances of the component eg:
> ```javascript
> value: function() { return {'X-Requested-With': 'XMLHttpRequest'} }
> ```

Reply:
ah- interesting- why wouldn't they be deep-cloned? nevertheless i'll update this